src/Integration.Vsix.UnitTests/SonarAnalyzer/SonarAnalyzerManagerForBoundTests.cs

```diff
@@ -18,12 +18,12 @@
 namespace SonarLint.VisualStudio.Integration.UnitTests.SonarAnalyzer
 {
     [TestClass]
-    public class SonarAnalyzerDeactivationManagerForBoundTests
+    public class SonarAnalyzerManagerForBoundTests
```

Comment:
That's not exactly the same. It will work in this case, because only one of the classes had an initialize method. But at the same time it makes the code harder to understand. I'll do the file renaming, but not use partial classes. Let's revisit this when the PR is ready for real review.